api/valor_api/logging.py

```diff
@@ -34,15 +34,32 @@ async def handle_request_validation_exception(
     return response
 
 
+async def handle_http_exception(
+    request: Request, exc: HTTPException
+) -> Union[JSONResponse, Response]:
+    if exc.status_code >= 500:
```

Comment:
Only logging status >= 500.  Let me know if you guys think this is reasonable.

Reply:
i think so!